node_modules/gh-users/lib/rest.js

```diff
@@ -82,7 +84,13 @@ GrassHopper.appRouter.on('get', '/api/users/:id', getUser);
  * @Return      {Me}                                        The current user
  */
 GrassHopper.appRouter.on('get', '/api/me', function(req, res) {
-    res.status(200).send(req.ctx.user);
+    var me = {
```

Comment:
Should we have a `getMe` function in the API that takes care of the required me processing?